src/libs/actions/Report.js

```diff
@@ -538,7 +587,10 @@ function addAction(reportID, text, file) {
                 },
             ],
             automatic: false,
-            sequenceNumber: newSequenceNumber,
+
+            // Use the client generated ID as a optimistic action ID
```

Comment:
NAB, but I'd love to stop truncating comments everywhere

Reply:
Oh hmm sorry not sure what you mean by this? But happy to make any changes.

Reply:
Ah I think I figured out what you mean. We actually added a lint rule that requires us to break a line after a certain length. But sometimes we are breaking much earlier than that.